metrics/lint/openapi-validator.go

```diff
@@ -0,0 +1,168 @@
+package linter
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"google.golang.org/protobuf/proto"
+)
+
+type Lint struct {
```

Comment:
Please add a comment here (above this struct) that explains where these definitions come from and link to their source - this might be code or documentation in the IBM linter project.

Reply:
Also, if this is specific to the IBM linter, could it be namespaced appropriately? 

Both this and the spectral linter support might be better in their own subpackages of metrics/lint